nixos/modules/services/web-apps/grocy.nix

```diff
@@ -24,6 +24,19 @@ in {
       '';
     };
 
+    nginx.securityHeaders = mkOption {
+      type = types.str;
+      default = ''
```

Comment:
This is not a good idea: as soon as one adds one other header, the other ones will be removed by default.

Reply:
Maybe the option should be `types.bool`. If false no headers will be included into the configuration. If true a set of security headers will be included into the configuration.